src/config.js

```diff
@@ -34,6 +34,9 @@ export const API_URLS = {
       case TargetPlatforms.JUNIT:
         url = `${this.server}/api/${version}/compiler/test`;
         break;
+      case TargetPlatforms.SWIFT_EXPORT:
+        url = `${this.server}/api/${version}/${TargetPlatforms.SWIFT_EXPORT.id}/compiler/translate?compiler=swift-export`;
```

Comment:
Why we use `compiler` is it based on jscode?

Reply:
Yeah, it's based on JS. Here's more details https://github.com/JetBrains/kotlin-compiler-server/pull/723

Reply:
But, you create for ex. iframe for executing, and so on:

[this.jsExecutor.reloadIframeScripts](jsLibs, this.getNodeForMountIframe(), targetPlatform);](https://github.com/JetBrains/kotlin-playground/blob/9be97466925f017ca888b0457c3d015d2964038a/src/executable-code/executable-fragment.js#L329)

<img width="1526" alt="Screenshot 2024-05-29 at 16 40 54" src="https://github.com/JetBrains/kotlin-playground/assets/242514/332a0b81-7fa7-47dd-9a4d-cd0eca883d88">